app/views/static_pages/home.html.erb

```diff
@@ -1,7 +1,7 @@
 <h1>￼Connecting businesses, communities and the tech talent pipeline</h1>
 
 <div class="links">
-  <%= link_to "Learn More", learn_more_index_path %>
+  <%= link_to "Learn More", learn_more_path %>
   <%= link_to "Sign Up", signup_path %>
 </div>
 <div class="map"></div>
```

Comment:
I believe we need to move the map to the locations page, based on the new layouts we received